hal_st/std_periph/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ install(TARGETS hal_st.std_periph EXPORT halStTargets)
 
 if("${TARGET_MCU_FAMILY}" STREQUAL stm32f0xx)
     target_link_libraries(hal_st.std_periph INTERFACE hal_st.atomic)
-    target_link_libraries(hal_st.std_periph INTERFACE hal_st.std_periph_stm32wbxx)
+    target_link_libraries(hal_st.std_periph INTERFACE hal_st.std_periph_stm32f0xx)
```

Comment:
What about linking to hal_st.std_periph_${TARGET_MCU_FAMILY}?

Reply:
Hmm... that doesn't sound too bad 😄